app/src/components/notifications/PageNotifications.vue

```diff
@@ -1,31 +1,34 @@
 <template>
-  <TmPage
-    data-title="My alerts"
-    :loading="$apollo.queries.notifications.loading"
-    :empty="notifications.length === 0"
-    :empty-title="`You don't have any notifications yet`"
-    :empty-subtitle="`Don't worry, they are on their way!`"
-  >
-    <template>
-      <EventList
-        :events="notifications"
-        :more-available="moreAvailable"
-        @loadMore="loadMore"
-      >
-        <template slot-scope="event">
-          <router-link :key="event.id" class="notification" :to="event.link">
-            <div class="content">
-              <img :src="event.icon" />
-              <div>
-                <h3 class="title">{{ event.title }}</h3>
+  <div class="notifications-container">
+    <h2>Notifications</h2>
+    <TmPage
+      data-title="My alerts"
+      :loading="$apollo.queries.notifications.loading"
+      :empty="notifications.length === 0"
+      :empty-title="`You don't have any notifications yet`"
+      :empty-subtitle="`Don't worry, they are on their way!`"
```

Comment:
```suggestion
      :empty-subtitle="`To start receiving notifications, all you have to do is use an address on any Lunie supported network. We'll take care of the rest!`"
```

Reply:
Looks good to me. What do you think, @jrmoreau ?